### ISSUE TITLE
Need to move camera position to a specific Marker or Position.

### ISSUE THREAD

**Issue by shaiful00000** [vietmap-company/vietmap-flutter-navigation#27]
Are there any way to move current camera position to a specific marker or Point?

**Comment by thanhdt1998** [comment]
The SDK has two functions to move the camera to a specific latlng:
```
  _mapController?.moveCamera(CameraUpdate.newLatLngZoom(LatLng(22.553147478403194, 77.23388671875), 14));  
  _mapController?.animateCamera(CameraUpdate.newLatLngZoom(LatLng(28.698791, 77.121243), 14));
``` 
You can get the marker latlng and pass it to this function, the SDK will move the map to the marker location.
If you need to move to the screen point position, please convert it to LatLng using this function:
```
  _mapController?.toLatLng(Point screenLocation)
```

**Comment by saiful00000** [comment]
Thanks for your replay. What is the type of _mapController? ?.

Well as I am using "NavigationView" widget and i am getting a controller which type is "MapNavigationViewController" in onMapCreated callback. 

and there is not method defined in MapNavigationViewController class.

Could you help me with this?

**Comment by thanhdt1998** [comment]
Oh, sorry for my mistake. At this time, the navigation does not allow to move the map programmatically. We will update `moveCamera` and `animateCamera` functions in version 3.1.0.

**Comment by saiful00000** [comment]
Thanks and waiting for the update.

**Comment by thanhdt1998** [comment]
Hi @saiful00000, we've published [v3.1.0-pre.1](https://pub.dev/packages/vietmap_flutter_navigation/versions/3.1.0-pre.1#useful-function). Please upgrade your dependency version and try again. 

If you encounter any errors or want to improve UX, feel free to create a new issue. Thank you.

**Comment by saiful00000** [comment]
Thanks. I truly appreciate your effort.
